### DISCUSSION TITLE
apisix和apisix-dashboard高版本依赖太多，解决各种依赖限制非常花时间

### DISCUSSION THREAD

**Discussion by cherishman2005** [apache/apisix#8760] [Q&A]
apisix和apisix-dashboard高版本依赖太多，解决各种依赖限制非常花时间

如：
* cypress包非常大，很多linux或windows环境 yarn install下载失败。
* apisix高版本编译： make deps基本是编译不过，低版本就比较顺利。

**Comment by attson** [comment]
https://github.com/apache/apisix/discussions/8757
同求解，我倒是看似编译过了，但是接口访问报 nil value. 搞不明白，好像就我有这样的问题似的。
